drivers/client_access_control/spec/requests/client_visibility_legacy_spec.rb

```diff
@@ -97,10 +101,11 @@
         expect(response).to have_http_status(200)
       end
       it 'can see window source client, but not non-window source client' do
-        get client_path(both_destination_client)
+        get rollup_client_path(both_destination_client, partial: :demographics)
```

Comment:
does this seem reasonable? We check the demographics rollup since I don't know that the main page has a list of source clients. The client names are not always visible due to permissions so check ids in a data attr.

Reply:
Yeah, I think this is a good solution, thanks!